package.json

```
@@ -85,17 +87,18 @@
     "remark-cli": "^9.0.0",
     "remark-preset-wooorm": "^8.0.0",
     "resolve-from": "^5.0.0",
+    "rollup": "^2.0.0",
+    "rollup-plugin-terser": "^7.0.0",
     "tape": "^5.0.0",
-    "tinyify": "^3.0.0",
     "xo": "^0.34.0"
   },
   "scripts": {
     "generate-constant-typings": "node script/generate-constant-typings",
     "generate-expressions": "node script/generate-expressions",
-    "generate-dist": "babel lib --out-dir dist --quiet --retain-lines --plugins babel-plugin-unassert,./script/babel-transform-undebug.js,./script/babel-transform-constants.js; node script/copy-dict.js; prettier dist --loglevel error --write",
-    "generate-size": "browserify . -p tinyify -s micromark -o micromark.min.js; gzip-size micromark.min.js; gzip-size --raw micromark.min.js",
+    "generate-dist": "babel lib --out-dir dist --quiet --retain-lines --plugins babel-plugin-unassert,./script/babel-transform-undebug.js,./script/babel-transform-constants.js; node script/copy-dict.js; prettier --loglevel error --write dist",
+    "generate-size": "rollup -c; gzip-size micromark.min.js; gzip-size --raw micromark.min.js",
```

```suggestion
    "generate-size": "rollup -c --silent; gzip-size micromark.min.js; gzip-size --raw micromark.min.js",
```

The default output by rollup is a bit more verbose than I’d like. Silent sounds a bit harsh, maybe there’s something in between, or --silent is good enough?